kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -150,26 +149,17 @@ def exec_foundry_to_k(
             modules.append(module)
             if claims_module:
                 _LOGGER.info(f'Produced claim module: {claims_module.name}')
-                claims_modules.append(claims_module)
+                claims.extend((claims_module.name, claim) for claim in claims_module.claims)
     _main_module = KFlatModule(
         main_module if main_module else 'MAIN', [], [KImport(mname) for mname in [_m.name for _m in modules] + imports]
     )
-    _spec_module = KFlatModule(
-        spec_module if spec_module else 'SPEC', [], [KImport(mname) for mname in [_m.name for _m in claims_modules]]
-    )
```

Comment:
What's the reason behind removing the spec module? Is it to separate the auto-generated rules from the hand-written claims?

Reply:
Just to reduce the output size since we aren't using the claims anymore to make the input specifications. The KCFGs are stored structurally (instead of textual claims), and then when we want to run the proof, the KCFG is written as a claim to a file for calling KProve on.